shippable.yml

```
@@ -31,9 +31,12 @@ build:
         - ./tests/ui/runtests.sh --headless --log-junit `pwd`/shippable/testresults --sso
         - ./vendor/phpunit/phpunit/phpunit -c ./tests/integration/phpunit.xml.dist --testsuite sso --log-junit `pwd`/shippable/testresults/xdmod-sso-integration.xml
         - mv ./configuration/portal_settings.ini.old ./configuration/portal_settings.ini
-        - if [ -s /var/log/xdmod/apache-error.log ]; then cat /var/log/xdmod/apache-error.log; false; fi
+        - [ ! -s /var/log/xdmod/apache-error.log ]
```

Shippable does not like my previous suggestion, but this should work:
```suggestion
        - test ! -s /var/log/xdmod/apache-error.log
```